cip25/rust/src/utils.rs

```diff
@@ -215,13 +610,21 @@ mod tests {
                 ChunkableString::from("src2"),
             ),
         ]);
-        let mut v2 = Data::new();
-        let mut v2_inner = BTreeMap::new();
-        v2_inner.insert(AssetNameV2::from(vec![0xCA, 0xFE, 0xD0, 0x0D]), details);
-        v2.insert(PolicyIdV2::from(vec![0xBA, 0xAD, 0xF0, 0x0D]), v2_inner);
-        let metadata = CIP25Metadata::new(LabelMetadata::new_label_metadata_v2(
-            LabelMetadataV2::new(v2),
-        ));
+        let policy_id_bytes = [
```

Comment:
Our old tests were using invalid policy IDs. This was never noticed since generating from the spec couldn't do things like say the text must be hex, only that it's text, etc. The bytes from the official .cddl also don't specify length for policy ids properly. Now that we're directly using `cml_chain::PolicyId` we ran into this so I updated the tests.